src/core/input/input_utils.hpp

```diff
@@ -17,7 +17,7 @@ namespace kb {
 	/** If event is a KeyPressed/KeyReleased event, updates keyPressed status. */
 	void checkKeyPressed(sf::Event event);
 
-#	define IS_KEY_PRESSED lif::kb:isKeyPressed
+#	define IS_KEY_PRESSED lif::kb::isKeyPressed
```

Comment:
Has this been broken until now? I admittedly haven't compiled the game on osx in years, so it wouldn't surprise me too much, but damn :|

Reply:
This never compiled due to undefined `IS_APPLE` macro.